core/src/main/java/org/apache/calcite/avatica/DriverVersion.java

```diff
@@ -96,8 +96,9 @@ public static DriverVersion load(
     int minorVersion = 0;
     int databaseMajorVersion = 0;
     int databaseMinorVersion = 0;
+    InputStream inStream = null;
     try {
-      final InputStream inStream =
+      inStream =
```

Comment:
Perhaps `try with resources` may be suitable here.

Reply:
Yes, you are right. I just checked https://calcite.apache.org/avatica/develop/avatica.html#download-source-build-and-run-tests, the required Java version is (JDK 1.8 or later, 1.9 preferred). Then try-with-resources is preferred here.